src-v2/Not/Not.Blazor/TM/NotChildren.razor

```diff
@@ -45,18 +47,25 @@ else
     public string NoItemsMessage { get; set; } = default!;
     [Parameter]
     public string UpdateRoute { get; set; } = default!;
+    [Parameter]
+    public string Class { get; set; } = default!;
```

Comment:
Do you think there is value in exposing parameters that aren't being used? It's not a hard thing to add a parameter if you need it at some point. And allowing unnecessary configuration can lead to overengineering and inconsistencies, both of which aren't ideal. Same goes for the `NoItemsMessage`.